### PR TITLE
add a "delegation lists" section

### PR DESCRIPTION
**Pull request by thomas-fossati**
Partially addresses #177

### CODE REVIEW

File: STAR-Delegation/draft-ietf-acme-star-delegation.md [hunk 1]
```diff
@@ -275,6 +275,32 @@ extended with a new `delegations` attribute:
 ~~~
 {: #fig-account-object title="Example Account object with delegations"}
 
+##### Delegation Lists
```
Comment (yaronf) [inline STAR-Delegation/draft-ietf-acme-star-delegation.md:278]: One level fewer please.

File: STAR-Delegation/draft-ietf-acme-star-delegation.md [hunk 2]
```diff
@@ -275,6 +275,32 @@ extended with a new `delegations` attribute:
...
+available to the NDC account ({{sec-delegation-objects}}).  The server MAY
+return an incomplete list, along with a Link header field with a `next` link
+relation indicating where further entries can be acquired.
+
+~~~
+HTTP/1.1 200 OK
+Content-Type: application/json
+Link: <https://acme.ido.example/acme/directory>;rel="index"
+Link: <https://acme.ido.example/acme/delegations/adFqoz?cursor=2>;rel="next"
+
```
Comment (yaronf) [inline STAR-Delegation/draft-ietf-acme-star-delegation.md:293]: The outer map is unnecessary IMHO, but it is in line with RFC 8555, so fine.

**Review by yaronf — CHANGES_REQUESTED** [review]

File: STAR-Delegation/draft-ietf-acme-star-delegation.md (hunk 2 above)
Comment (yaronf) [reply STAR-Delegation/draft-ietf-acme-star-delegation.md:293]: I stand corrected: https://softwareengineering.stackexchange.com/questions/286293/whats-the-best-way-to-return-an-array-as-a-response-in-a-restful-api

File: STAR-Delegation/draft-ietf-acme-star-delegation.md (hunk 1 above)
Comment (thomas-fossati) [inline STAR-Delegation/draft-ietf-acme-star-delegation.md:278]:
```suggestion
#### Delegation Lists
```

**Pull request merged**